nengo_gui/static/hotkeys.js

```diff
@@ -86,6 +86,19 @@ Nengo.Hotkeys = function () {
                 Nengo.ace.update_trigger = true;
                 ev.preventDefault();
             }
+            // toggle auto-hide of console with TODO: pick a good shortcut
+            if (ctrl && ev.shiftKey && key == 'h') {
+                Nengo.ace.auto_hide_console = !Nengo.ace.auto_hide_console;
+                if (Nengo.ace.auto_hide_console == false) {
+                    Nengo.ace.console_height = document.getElementById('vmiddle').clientHeight / 4.0;
```

Comment:
just from usability, i changed it because i found that i was resizing it to be larger every time i displayed it. with the not auto display the height doesn't change to automatically display the whole error message, and so i found there wasn't really any error message where i could read the whole thing at the default height